content/documentation/device-examples/components/Camera.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Camera
+weight: 210
+tags: ["Camera"]
+---
+### Description
+The [Camera](https://github.com/Pi4J/pi4j-example-components/tree/Dev-Arcade/src/main/java/com/pi4j/components/components) (src/main/java/com/pi4j/components/components) is a template class, that you can use in your own Java-project.
```

Comment:
src/main/java/com/pi4j/components/components use the same link as in all other files. I will ask tomorrow if we should change the link